.github/workflows/release.yml

```diff
@@ -0,0 +1,100 @@
+name: 'release'
+on:
+  push:
+    tags:
+      - 'v*'
+
+env:
+  # .goreleaser.docker.yaml reads DOCKER_REPO
+  DOCKER_REPO: 'us-docker.pkg.dev/abcxyz-artifacts/docker-images/pmap'
+  WIF_PROVIDER: 'projects/921163060412/locations/global/workloadIdentityPools/github-pool-3b39/providers/github-provider'
+  WIF_SERVICE_ACCOUNT: 'pmap-3b39-ci-sa@pmap-ci.iam.gserviceaccount.com'
+  DEV_PROJECT_ID: 'pmap-dev'
+  DEV_REGION: 'us-central1'
+
+# Don't cancel in progress since we don't want to have half-baked release.
+concurrency: '${{ github.workflow }}-${{ github.head_ref || github.ref }}-release'
+
+jobs:
+  ci:
+    uses: 'abcxyz/pmap/.github/workflows/ci.yml@main' # ratchet:exclude
```

Comment:
I'm not sure we want this? Don't we want "needs" instead?

Reply:
JVS has the same pattern here: https://github.com/abcxyz/jvs/blob/main/.github/workflows/release.yml#L17-L19.

The purpose here is to run CI before starting the release process. The line for `needs: [ci]` is a few more lines below at [here](https://github.com/abcxyz/pmap/blob/sailorlqh/add_release_wf/.github/workflows/release.yml#L24).